Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCrowdStrikeFalconHost.yaml

```diff
@@ -0,0 +1,379 @@
+                                    "-", 
+                                    (temp_isSrcMatch and temp_isDstMatch),
+                                    "Both", 
+                                    temp_isSrcMatch,
+                                    "SrcIpAddr",
+                                    temp_isDstMatch,
+                                    "DstIpAddr",
+                                    "No match" 
+                                )        
+          | where ASimMatchingIpAddr != "No match"
+          | extend deviceIp = iff(hostName matches regex "(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", hostName, "")
+          | extend 
+              hostName = iff(isempty(deviceIp), hostName, ""),
+              AdditionalFields = bag_pack(
+                        "networkProfile", networkProfile,
+                        "ruleDescription", ruleDescription,
+                        "ruleGroupName", ruleGroupName,
+                        "cmdLine", cmdLine
+                    ),
+              NetworkIcmpCode = icmpCode
+          | invoke _ASIM_ResolveDvcFQDN('hostName')
+          | invoke _ASIM_ResolveNetworkProtocol('protocol')
+          | extend NetworkIcmpType = _ASIM_LookupICMPType('icmpType')
+          | project-rename
+              DvcId = deviceId,
+              DvcIpAddr = deviceIp,
+              EventOriginalSubType = eventType,
+              NetworkRuleName = ruleName
+          | extend
+              Rule = NetworkRuleName,
+              Dvc = coalesce(DvcId, DvcHostname, DvcIpAddr);
+      let networkaccessdata = alldata
+          | where DeviceEventClassID has "Network Access In A Detection Summary Event"
+          | lookup ActionLookup on EventOutcome
+          | where ((array_length(dvcaction) == 0) or DvcAction has_any (dvcaction))
+          | where ((eventresult == "*") or EventResult == eventresult)
+          | extend
+              temp_isSrcMatch = has_any_ipv4_prefix(SourceIP, src_or_any), 
+              temp_isDstMatch = has_any_ipv4_prefix(DestinationIP, dst_or_any)
+          | extend ASimMatchingIpAddr = case(
+                                    array_length(src_or_any) == 0 and array_length(dst_or_any) == 0,
+                                    "-", 
+                                    (temp_isSrcMatch and temp_isDstMatch),
+                                    "Both", 
+                                    temp_isSrcMatch,
+                                    "SrcIpAddr",
+                                    temp_isDstMatch,
+                                    "DstIpAddr",
+                                    "No match" 
+                                )        
+          | where ASimMatchingIpAddr != "No match"
+          | parse-kv AdditionalExtensions as (CSMTRPatternDisposition: string) with (pair_delimiter=";", kv_delimiter="=")
+          | invoke _ASIM_ResolveSrcFQDN('DestinationHostName')
```

Comment:
is this is Destination or Src field?

Reply:
it is the source Device Hostname where the CrowdStrike agent is installed and the event is reported.

Reply:
@niralishah-crest @jayeshprajapaticrest 
Field name says destination 'DestinationHostName'? Hope you have validate this.

Reply:
it is the Source Device Hostname where the CrowdStrike agent is installed and the event is reported.